mem.c:439:14: warning: variable ‘xms_largest_free_kb’ set but not used [-Wunused-but-set-variable]
  439 |     uint32_t xms_largest_free_kb = 0;
      |              ^~~~~~~~~~~~~~~~~~~

--- a/ci_prereq.sh
+++ b/ci_prereq.sh
@@ -5,6 +5,6 @@ sudo apt install -y \
   devscripts \
   equivs
 
-sudo add-apt-repository ppa:stsp-0/thunk-gen
-sudo add-apt-repository ppa:stsp-0/dj64
+sudo add-apt-repository -y ppa:stsp-0/thunk-gen || (sleep 5 && sudo add-apt-repository -y ppa:stsp-0/thunk-gen)
+sudo add-apt-repository -y ppa:stsp-0/dj64 || (sleep 5 && sudo add-apt-repository -y ppa:stsp-0/dj64)
 mk-build-deps --install --root-cmd sudo

